source/_integrations/zha.markdown

```diff
@@ -16,7 +16,11 @@ featured: true
 ---
 
 [Zigbee Home Automation](http://www.zigbee.org/zigbee-for-developers/applicationstandards/zigbeehomeautomation/)
-integration for Home Assistant allows you to connect many off-the-shelf Zigbee based devices to Home Assistant, using one of the available Zigbee radio modules compatible with [zigpy](https://github.com/zigpy/zigpy) (an open source Python library implementing a Zigbee stack, which in turn relies on separate libraries which can each interface a with Zigbee radio module a different manufacturer).
+integration for Home Assistant allows you to connect many off-the-shelf Zigbee based devices directly to Home Assistant using one of the several available Zigbee radio modules hardware compatible with [zigpy](https://github.com/zigpy/zigpy). 
+
+Zigpy is an open source Python library implementing a Zigbee stack, it functions as a middleware which in turn relies on separate Python libraries which each acts a wrapper to abstract the unique interfaces of Zigbee radio modules from different manufacturer).
```

Comment:
Can we rewrite this in a way a normal user could understand this better?
This sentence is pretty long and full of technical words, most users would have a hard time reading this.

Reply:
Suggestions on better phrasing? My original thinking was that most Home Assistant users today are very technical and that the introduction text is also for new developers too, not just end-users.

Reply:
Developers have their own website, with the doc website the end users are always central and especially people who start at 0